docs/releases/v9_3.md

```diff
@@ -12,7 +12,7 @@ The **April 14th, 2024** release of **DietPi v9.3** comes with an own DietPi APT
 
 #### Enhancements
 
-- **General** :octicons-arrow-right-16: We added an own APT repository at <https://dietpi.com/apt/> for own software and kernel/firmware builds. This simplifies updates for software packages like [Squeezelite](../software/media.md#squeezelite), [vaultwarden](../software/cloud.md#vaultwarden) and [Amiberry](../software/gaming.md#amiberry), as well as allows us to provide own up-to-date kernel and bootloader packages, applied via regular `apt upgrade`. A goal is also to become independent of some 3rd party repositories, which are regularly affected by network errors or provide package updates too infrequently.
+- **General** :octicons-arrow-right-16: We added an own APT repository at <https://dietpi.com/apt/> for own software and kernel/firmware builds. This simplifies updates for software packages like [Squeezelite](../software/media.md#squeezelite), [vaultwarden](../software/cloud.md#vaultwarden) and [Amiberry](../software/gaming.md#amiberry), as well as allowing us to provide own up-to-date kernel and bootloader packages, applied via regular `apt upgrade`. A goal is also to become independent of some 3rd party repositories, which are regularly affected by network errors or provide package updates too infrequently.
```

Comment:
I think this was one correct already: "This simplifies ..., as well as allows". There is no "is" for present progressive 🙂.